OpenRA.Game/ObjectCreator.cs

```diff
@@ -54,7 +55,14 @@ public ObjectCreator(Manifest manifest, FileSystem.FileSystem modFiles)
 				Assembly assembly;
 				if (!ResolvedAssemblies.TryGetValue(hash, out assembly))
 				{
-					assembly = Assembly.Load(data);
+					using (Stream stream = modFiles.Open(path))
```

Comment:
Please make this reuse the `modFiles.Open` above instead of loading another stream of the data.